### DISCUSSION TITLE
Grafana - is there a way to disable the graph refresh icon?

### DISCUSSION THREAD

**Discussion by IceDragonix** [grafana/grafana#43010] [Dashboards, Panels & Visualizations]
Hello,

I'm running Grafana on Raspberry Pi 4B (4 GB). I'm using it to monitor selected parameters of my main computer (via HWiNFO and Prometheus).

Raspberry Pi is not a speed demon. The dashboard is set to refresh every 5 seconds. Each time it refreshes there is a spinning arrows animation on each of the graphs (please see the attached picture). It's distracting and not needed in my use case. 

Is there a way to disable the animated icon?

![Refresh](https://user-images.githubusercontent.com/70708661/145712944-cc31bbf4-37b3-450b-ace5-0c64c4ec3080.jpg)

**Comment by michaelmcdonald** [comment]
Almost a year old post, but I thought I would bump it since I have a similar issue / annoyance. Closest "solution" I've come across (though yet to implement) is the following:

https://community.grafana.com/t/how-to-disable-refresh-indicator-in-top-right-corner-of-each-panel/52114/3

That post (that I directly link to) and the one below (discussing the "Boom" theme) seem to be the only method. If someone has a better solution I would love it hear it!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
